data/abilities.ts

```diff
@@ -2875,6 +2875,11 @@ export const Abilities: import('../sim/dex-abilities').AbilityDataTable = {
 				}
 			}
 		},
+		onAnySwitchIn(pokemon) {
```

Comment:
I feel like this should not be tied to Neutralizing Gas specifically. I'm sure there's some Baton Pass Gastro Acid shenanigans you can do to cause the same thing to happen. Wouldn't it be better to just delete the effectState for Protean/Libero/Embody Aspect when the Pokemon leaves the field? I don't see why we're removing it on switchin in the first place.